tardis/io/model/stella.py

```diff
@@ -48,13 +48,14 @@ def read_stella_model(fname):
                 header_re_match = header_re[i].match(line)
 
                 metadata[HEADER_RE_STR[i][1]] = header_re_match.group(1)
-            if line.strip().startswith("mass of cell"):
-                column_names_raw = re.split(r"\s{3,}", line.strip())
-                break
-        else:
-            raise ValueError(
-                '"mass of cell" is required in the Stella input file to infer columns'
-            )
+            elif i + 1 == DATA_START_ROW:
```

Comment:
Why are you checking i+1 in a loop? Won't the loop reach i+1 in the next iteration anyway?